projects/plugins/jetpack/modules/widgets/rsslinks-widget.php

```diff
@@ -221,6 +221,10 @@ public function form( $instance ) {
 	 * @param array  $args Widget arguments.
 	 */
 	private function rss_link( $type, $args ) {
+		$link_text     = null;
+		$rss_type      = null;
+		$subscribe_to  = null;
+		$link_contents = null;
```

Comment:
I'm tempted to move `$link_contents` down to just before line 253.

Reply:
I agree it's more readable; done!